lms/static/scripts/frontend_apps/components/BasicLTILaunchApp.js

```diff
@@ -97,7 +95,7 @@ export default function BasicLTILaunchApp() {
 
   // Content is ready to show if we've resolved a contentURL or configuration
   // for a VitalSource document is present
-  const contentReady = !!(contentURL || vitalSourceConfig);
+  const contentReady = !!contentURL;
```

Comment:
In the backend code changes, there's a helpful comment about VitalSource not using Via, but otherwise behaving the same. I wonder if it might be possible to add a comment in front-end code, perhaps around where `contentURL` is assigned...if there could be any confusion?

Reply:
I've updated the comments for `contentURL` to explain what it is and where it comes from.